CommentAgent.py

```diff
@@ -199,11 +199,14 @@ def init_GPT(self) -> None:
         self.GPT.add_message("system", GPTsettings.SYSTEM_PROMPT)
         self.add_pr_messages()
 
-    def add_custom_instructions(self) -> None:
-        file_contents = self.GPT.get_file_contents("agent-settings/README.md")
+    def get_custom_instructions(self) -> str:
+        """Retrieves custom instructions for the repository from a markdown file if it exists."""
+        file_contents = self.GH.get_file_contents("agent-settings/README.md")
         if file_contents != "Could not retrieve file contents.":
-            custom_instructions_prompt = f"""The following are custom instructions for this repository:\n{file_contents}"""
-            self.GPT.add_message("user", custom_instructions_prompt)
+            custom_instructions_prompt = f"""The Repository has some prdefined rules for reviewers:\n{file_contents}"""
```

Comment:
Consider revising the docstring and string formatting to improve clarity and correctness. Here's a suggested change:
```suggestion
        def get_custom_instructions(self) -> str:
            """Retrieves custom instructions for the repository from a markdown file if it exists."""
            file_contents = self.GH.get_file_contents("agent-settings/README.md")
            if file_contents != "Could not retrieve file contents.":
                custom_instructions_prompt = "The repository has predefined rules for reviewers:
{file_contents}"
                return custom_instructions_prompt
            else:
                return ""
```
        Also, consider handling the potential exception when file retrieval fails for reasons other than the file not existing.